.env.development

```diff
@@ -1,7 +1,2 @@
-# Default admin/district authorization for development
 REACT_APP_DHIS2_BASE_URL="http://localhost:8080/dhis"
```

Comment:
I don't think this should be necessary, the platform will present a login dialog if this env var isn't provided